app/models/building.rb

```diff
@@ -0,0 +1,8 @@
+class Building < ActiveRecord::Base
+  has_many :tenants, class_name: User.name
```

Comment:
How will we handle this once buildings have many apartments, apartments have many tenants, and tenants have one user? Just thinking ahead here to the refactor of User.